lib/mongo/server.rb

```diff
@@ -649,14 +659,14 @@ def clear_description
     #   service id only.
     #
     # @api private
-    def clear_connection_pool(service_id: nil)
+    def clear_connection_pool(service_id: nil, interrupt_in_use_connections: false)
```

Comment:
Documenting comment for `interrupt_in_use_connections` is missing.

Reply:
done